zebrad/src/commands/start.rs

```diff
@@ -22,15 +22,13 @@ use crate::config::ZebradConfig;
 use crate::{components::tokio::TokioComponent, prelude::*};
 use abscissa_core::{config, Command, FrameworkError, Options, Runnable};
 use color_eyre::Report;
-use eyre::{eyre, WrapErr};
-use futures::{
-    prelude::*,
-    stream::{FuturesUnordered, StreamExt},
-};
-use std::collections::BTreeSet;
-use tower::{buffer::Buffer, service_fn, Service, ServiceExt};
+use futures::stream::FuturesUnordered;
+use std::collections::{BTreeSet, HashSet};
+use tower::{buffer::Buffer, service_fn};
 use zebra_chain::{block::BlockHeaderHash, types::BlockHeight};
 
+mod sync;
+
 // genesis
 static GENESIS: BlockHeaderHash = BlockHeaderHash([
```

Comment:
Is this the main net genesis block hash?

Is the genesis block for the test net different?

Reply:
:grimacing:

Reply:
Yes, the genesis hash is network-dependent, but there shouldn’t need to be a constant here since it can be fetched from the state

Reply:
Ok, so we'll need to pass the network to the state, and then it can initialise itself with the correct genesis block. (But until that's implemented, I think a constant is fine.)